### ISSUE TITLE
Bursa aspiration: crystals present

### ISSUE THREAD

**Issue by MickeySegal** [obophenotype/human-phenotype-ontology#10820]
**Preferred term label:**

**Synonyms**
Uric acid; monosodium urate crystals; calcium pyrophosphate dihydrate crystals in bursa fluid

**Definition (free text, please give PubMed ID)**

**Parent term (use hpo.jax.org/app)**

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**
Olecranon bursitis
Prepatellar bursitis

**Your nano-attribution (ORCID)**

**Comment by pnrobinson** [comment]
@MickeySegal 
Adding new term: 
- ID: HP:6001156
- label: Synovial crystals
- parents: [Abnormal synovial bursa morphology (HP:0025231)](https://hpo.jax.org/app/browse/term/HP:0025231)
- pmid: PMID:21285711
- def: The presence of microscopic crystalline structures in the synovial fluid (the lubricating fluid within joint cavities).
- comment: Synovial fluid analysis is a simple procedure requiring only a microscope fitted with polarized filters and a first-order red compensator. Synovial fluid is examined fresh without centrifugation, fixing or staining. Monosodium urate crystals responsible for gout and calcium pyrophosphate dehydrate crystals responsible for the related arthritis are of primary importance when performing a crystal search in synovial fluid in a clinical setting.
- orcid: orcid:0000-0002-0736-9199
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/10820
- synonyms: Crystals in synovial fluid(NONE)|Bursa aspiration: crystals present(NONE)

**Comment by MickeySegal** [comment]
We need to clarify both your term and our term in order to distinguish between bursa and synovium.  There is some discussion of this distinction at https://byjus.com/biology/difference-between-bursa-and-synovial-fluid/

I clarified our term to be "Non-synovial bursa aspiration: crystals present" to keep it separate from https://hpo.jax.org/browse/term/HP:6000248.  Your term here of "Synovial crystals" would be a synonym of HP:6000248.  Here it would be better to use "Non-synovial bursa crystals".

**Comment by pnrobinson** [comment]
reopening. I was not aware of nonsynovial bursa crystals, let's try to find a PMID with a precise definition!

**Comment by MickeySegal** [comment]
These may be helpful:

https://www.ncbi.nlm.nih.gov/books/NBK554617/

https://www.ncbi.nlm.nih.gov/sites/books/NBK557508/

**Comment by pnrobinson** [comment]
@MickeySegal 
Coming back to this, can you clarify whether the intention of your new term was to denote crystals in fluid within the bursa fluid. The bursa is a potential space that normally does not have appreciable fluid within it. This was what I was trying to say, but I have renamed the term "Synovial fluid crystals". I think this is also what your references were saying, e.g., "Needle aspiration of bursal fluid is crucial in differentiating between causes. ". Please close this issue if we are in agreement, otherwise please clarify!

**Comment by MickeySegal** [comment]
Synovial fluid and bursal fluid are different.  

We have several bursal findings in the pipeline, dealing with WBC, gram stain and crystals.  Although the bursa doesn't normally have much fluid, it does when there is bursitis, and that is when you do an aspiration.

**Comment by pnrobinson** [comment]
You are correct, I was not thinking!

I have added new parentages and named this term
HP:6000248
Bursa fluid crystals
syn: Bursa aspiration: crystals present; Crystals in bursa fluid
def: The presence of crystals (microscopic solids that form distinct lattices, with definable fixed angles, faces, walls) in the bursa fluid.
Comment: Bursa fluid is found inside bursae -small, fluid-filled sacs located around joints, especially at points of friction (e.g., between bone and tendon or skin). This feature can be ascertained by aspiration of the bursa fluid.
